requirements.txt

```diff
@@ -6,8 +6,8 @@ scipy>=0.18
 tabulate>=0.7
 scikit-learn<1.0
 tqdm
-tensorflow>=2.3.1,<2.6.0
-tensorflow-addons==0.13.0
+tensorflow>=2.3.1,<=2.7.0
```

Comment:
Might be better to use `<2.8` here. Hypothetically, if they release a 2.7.1, we probably want to consider that supported.

Reply:
Good idea -- done.